src/constants/properties.ts

```diff
@@ -0,0 +1,23 @@
+const DEFAULTS = {
```

Comment:
Can we call this file `properties.constant.ts` to be consistent with changes @IgnatG will be merging soon please?